content/en/faq.md

```diff
@@ -7,22 +7,154 @@ position: 900
 
 This FAQ is intended to go as in depth as possible for anyone using sigstore. 
 
-_[Looking for general FAQs?](https://www.sigstore.dev/how-it-works)_
+## General
 
-## Why is Rekor centralized?
+### "What security checks do you use internally?
+ 
+We’ve adopted a security disclosures and response policy to make sure we can responsibly handle critical issues. We have an initial Security Response Committee, who for each vulnerability reported will coordinate to create the fix and release, and keep the committee looped in. The policy in full is here: <https://github.com/sigstore/.github/blob/main/SECURITY.md>.
 
-- Well, why not? There's no need for a distributed source of transparency. There can be multiple points of transparency which only adds more sources of security guarantee, not less. An entity can post to as many Rekors as they want and inform users of where they post. We do encourage folks to use common public instances, but we don't seek to enforce this. We do plan to look to produce a gossip protocol, for those that desire a more decentralised model (if the demand is shown).
+### How does Sigstore integrate in-toto?
 
-## How do I verify downloaded code?
+Sigstore is co-led by Santiago Torres-Arias, the lead of the in-toto project, and the two efforts are complementary. Sigstore supports and uses in-toto in a number of ways. Cosign can be used to generate and verify in-toto attestations and Rekor can store those attestations. in-toto also supports the use of PKI semantics for signing metadata which will be used in future to support Fulcio for developer key management.
+
+### What is the relationship between Sigstore and TUF?
+
+Sigstore's root of trust for Fulcio and Rekor are distributed using The Update Framework [TUF](https://theupdateframework.io).  The releases for Cosign, Rekor, and Fulcio are also signed by a key distributed with TUF.  The projects solve a similar problem of signing and verifying software that will be distributed via a repository. Sigstore provides key signing using multiple techniques, including without requiring developers to manage keys by using OIDC.  In contrast, TUF typically requires keys to be managed by developers, but addresses a threat model where all online servers and keys could be compromised.  The projects are collaborating and can be used cooperatively.  More guidance on this will be forthcoming as the communities collaborate on their first large co-deployments.
+
+## Cosign
+
+### Why not use Notary v2?
```

Comment:
Maybe this can just be about what are the differences between Notary and Cosign vs. why not use an alternate tool?

Reply:
This is actually a long discussion.  So I added a quick sentence about differences and pointed to the page that discusses the issue in-depth.